recipes-config/systemd-resource-control/systemd-resource-control.bb

```diff
@@ -27,6 +28,7 @@ do_install_append() {
 
     # Install overrides for services
     install -d ${D}${systemd_unitdir}/system/neptune.service.d/
+    install -m 0644 ${WORKDIR}/service.bluetooth.conf ${D}${systemd_unitdir}/system/connman.service.d/
```

Comment:
Destination directory looks wrong.

If you do not know about it already you can verify overrides with `systemctl cat`. Comments with file the lines comes from will be added in output.

Reply:
Unresolving. `service.bluetooth.conf` still overrides `connman.service` instead of `bluetooth.serivce` it looks like to me. Or what am I missing?

Reply:
`service.bluetooth.conf `shall be installed into folder `bluetooth.service.d/`, not `connman.service.d/`

Reply:
I'm wondering if it would be better that the script is able to search for all files in the folder `services `which matches the syntax `service.*.conf `and installs them into `*.service.d/`.